Client/Frontend/Home/RecentlyClosedTabsPanel.swift

```diff
@@ -35,6 +35,7 @@ class RecentlyClosedTabsPanel: UIViewController, HomePanel {
 
     fileprivate lazy var historyBackButton: HistoryBackButton = {
         let button = HistoryBackButton()
+        button.accessibilityIdentifier = "goBackFromRecentlyClosedHistory"
         button.addTarget(self, action: #selector(RecentlyClosedTabsPanel.historyBackButtonWasTapped), for: .touchUpInside)
```

Comment:
@npark-mozilla Found this one line solution to add the identifier for this case, better than the coordinates approach, right?

@farhanpatel would this acc.ID addition be fine for you too?
Thanks!